keel-lemur/src/main/kotlin/com/netflix/spinnaker/config/LemurConfiguration.kt

```diff
@@ -0,0 +1,52 @@
+package com.netflix.spinnaker.config
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.keel.lemur.LemurService
+import okhttp3.HttpUrl
+import okhttp3.HttpUrl.Companion.toHttpUrlOrNull
+import okhttp3.OkHttpClient
+import org.springframework.beans.factory.BeanCreationException
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.http.HttpHeaders.AUTHORIZATION
+import retrofit2.Retrofit
+import retrofit2.converter.jackson.JacksonConverterFactory
+
+@Configuration
+@ConditionalOnProperty("lemur.base-url")
+class LemurConfiguration {
+  @Bean
+  fun lemurEndpoint(
+    @Value("\${lemur.base-url}") lemurBaseUrl: String
```

Comment:
I thought the prefered pattern for this stuff is to create a `LemurConfig` class with the properties in it, and wire that in via the `@EnableConfigurationProperties` annotation. I think it would be good to switch to that format so that we don't have the `@Value` annotations all over the place here.

Reply:
+1.

Reply:
Sure. I guess I felt like there was maybe a threshold of a number of properties where that became worthwhile, but I'm happy to do it.